tests/test_production.py

```diff
@@ -53,11 +54,22 @@ def build_config(override, basedir):
     return config
 
 
+async def _get_github_token():
```

Comment:
We may end up tweaking this to make testing locally easier, but I think this works for now.